src/__tests__/Search/SearchView.test.tsx

```diff
@@ -7,9 +8,22 @@ import { renderWithRouter, store } from '../utils/setupTests';
 import { screen } from '../utils/test-utils';
 
 describe('Search View', () => {
-  it('render correctly when there are no results', () => {
+  it('render correctly when there are no results', async () => {
+    const { testData } = getTestData();
+    global.fetch = jest.fn(() =>
+      Promise.resolve({
+        json: () => ({
+          results: testData,
+        }),
+      }),
+    ) as jest.Mock;
+
```

Comment:
without mocking the call to `fetchRecentRevisions` that occurs on mount of SearchView, the snapshot was displaying an error because fetch returned no results